backend/src/main/java/me/performancereservation/domain/refund/RefundService.java

```diff
@@ -0,0 +1,140 @@
+package me.performancereservation.domain.refund;
+
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import me.performancereservation.domain.performance.entities.Performance;
+import me.performancereservation.domain.refund.dto.RefundDetailResponse;
+import me.performancereservation.domain.refund.dto.RefundRequest;
+import me.performancereservation.domain.refund.dto.RefundResponse;
+import me.performancereservation.domain.refund.enums.RefundStatus;
+import me.performancereservation.domain.reservation.Reservation;
+import me.performancereservation.domain.reservation.ReservationRepository;
+import me.performancereservation.domain.reservation.enums.ReservationStatus;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class RefundService {
+
+    private final RefundRepository refundRepository;
+    private final ReservationRepository reservationRepository;
+
+    // refundRequest를 받고 Refund를 생성하여 저장
+    public Long save(RefundRequest refundRequest) {
+        // 이미 같은 예약ID인 Refund가 존재한다면 예외 던짐 (이미 환불신청된 이력 있음)
+        if (refundRepository.findRefundByReservationId(refundRequest.getReservationId()).isPresent()) {
+            throw new AppException(ErrorCode.DUPLICATE_REFUND_ERROR, ErrorType.DOMAIN);
+        }
```

Comment:
이미 ErrorCode에 AppException 인스턴스를 생성하는 로직이 있어서

```suggestion
        // 이미 같은 예약ID인 Refund가 존재한다면 예외 던짐 (이미 환불신청된 이력 있음)
        if (refundRepository.findRefundByReservationId(refundRequest.getReservationId()).isPresent()) {
            throw ErrorCode.DUPLICATE_REFUND_ERROR.domainException("로그로 확인할 메시지");
        }
```

이런식으로 작성해주시면 됩니다! domainException()을 호출하면 new AppException이 돼요

Reply:
반영하여 수정완료했습니다!